ceno_zkvm/benches/fibonacci.rs

```diff
@@ -0,0 +1,87 @@
+use std::{
+    fs,
+    path::PathBuf,
+    time::{Duration, Instant},
+};
+
+use ceno_emul::{CENO_PLATFORM, Platform, Program, WORD_SIZE};
+use ceno_zkvm::{
+    self,
+    e2e::{run_e2e_gen_witness, run_e2e_proof},
+};
+use criterion::*;
+
+use goldilocks::GoldilocksExt2;
+use mpcs::BasefoldDefault;
+
+criterion_group! {
+  name = op_add;
```

Comment:
Needs renaming.

Reply:
nice catch and fixed https://github.com/scroll-tech/ceno/pull/690/commits/19b6164b62d09cc2ca094e21c82d5c3b5cfd6183